EIPS/eip-x.md

```diff
@@ -60,6 +61,14 @@ Requests MUST NOT skip step 2 and be automatically claimed, due to the ambiguity
 
 Note that for redemption requests, whether yield still accrues on shares that are pending a redemption request, or only on shares that are pending a redemption request and have not been executed, is up to the Vault implementation.
 
+Cancellation requests also go through the same Pending, Claimable, and Claimed stages. An example lifecycle for a cancelled deposit request is visualized in the table below. 
```

Comment:
LGTM!

Reply:
I'd nit the table and use previewRedeem instead of redeem on the claimed state transition value